extensions/ql-vscode/src/contextual/templateProvider.ts

```diff
@@ -91,6 +86,11 @@ export class TemplateQueryDefinitionProvider implements DefinitionProvider {
           (src, _dest) => src === uriString,
         );
       },
+      {
+        location: ProgressLocation.Notification,
```

Comment:
Could you remove the location from here (due to the default) or is there something I am missing?

Reply:
Yep, we can definitely remove these.